### PR TITLE
add enableAapt2 to gradle.props

### PR DESCRIPTION
**Pull request by IsaacHardy**
## Creator

- [x] Ensure there are no new warnings (in app AND in storybook)
- [x] Set a relevant reviewer

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review the changelog entry

### CODE REVIEW

File: mobile/android/app/release/output.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"outputType":{"type":"APK"},"apkInfo":{"type":"MAIN","splits":[],"versionCode":6,"versionName":"0.3.0","enabled":true,"outputFile":"app-release.apk","fullName":"release","baseName":"release"},"path":"app-release.apk","properties":{}}]
```
Comment (burkeshartsis) [inline mobile/android/app/release/output.json]: I think this file and the entire `/release` directory can be removed.

File: mobile/android/gradle.properties [hunk 2]
```diff
@@ -18,3 +18,4 @@
 # org.gradle.parallel=true
 
 android.useDeprecatedNdk=true
+org.gradle.configureondemand=true
```
Comment (burkeshartsis) [inline mobile/android/gradle.properties]: I do not believe we need this anymore.

File: mobile/appcenter-pre-build.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```
Comment (burkeshartsis) [inline mobile/appcenter-pre-build.sh]: can this file live in `/scripts`

Comment (IsaacHardy) [reply mobile/appcenter-pre-build.sh]: No it cannot.

File: mobile/android/gradle.properties (hunk 2 above)
Comment (IsaacHardy) [reply mobile/android/gradle.properties]: You might be right.

File: mobile/android/app/release/output.json (hunk 1 above)
Comment (IsaacHardy) [reply mobile/android/app/release/output.json]: No idea where this came from 😬

Comment (burkeshartsis) [reply mobile/android/app/release/output.json]: It's generated when you create a release build.

**Pull request merged**